https://bugs.gentoo.org/347489

Separate LDFLAGS and LDLIBS so that local lib directory can take precedence
over any system default lib dirs in LDFLAGS.

--- iproute2-2.6.35/Makefile
+++ iproute2-2.6.35/Makefile
@@ -33,7 +33,9 @@ CCOPTS = -D_GNU_SOURCE -O2 -Wstrict-prototypes -Wall
 CFLAGS = $(CCOPTS) -I../include $(DEFINES)
 YACCFLAGS = -d -t -v
 
-LDLIBS += -L../lib -lnetlink -lutil
+LDLIBS += -lnetlink -lutil
+
+LDFLAGS := -L../lib $(LDFLAGS)
 
 SUBDIRS=lib ip tc misc netem genl
 
